File: Tienda_Pyton/Maquillaje.py
from Producto import Producto
class Maquillaje(Producto):
    ID = 200

    def __init__(self, nombre, precio, fecha_importacion, stock, color):
        super().__init__(nombre, precio, fecha_importacion, stock)
        self.color = color
        self.id = Maquillaje.ID
        Maquillaje.ID += 1

    def get_id(self):
        return self.id

    def obtener_informacion_con_color(self):
        return super().obtener_informacion() + ", color " + self.color + " ID: " + str(self.id)

File: Tienda_JAVA/src/javaapplication45/Maquillaje.java

package javaapplication45;

public class Maquillaje extends Producto {
    private static int ID = 200;
    int id;
    private String color;
    
    public Maquillaje( String nombre, double precio, String fechaImportacion, int stock, String color){
        super( nombre,  precio,  fechaImportacion,  stock);
        this.color = color;
        this.id=ID;
        ID++;
        
    }

    public int getId() {
        return id;
    }
    
    public String ObtenerInformaciopnconColor(){
    return super.obtenerInformacion() +", color "+ color+ "ID: "+id;
}
}
